src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountTests.fs

```diff
@@ -0,0 +1,115 @@
+module BankAccountTests
+
+open System
+open Deciders.BankAccount
+open Deciders.BankAccountPrimitives
+open FsCheck
+open Ogooreck.BusinessLogic
+open Deciders.BankAccountDecider
+open FsCheck.Xunit
+
+let random = Random()
+let now = DateTimeOffset.UtcNow
+
+let spec =
```

Comment:
if this is statefu; and/or risks being stateful, also maybe not a single static instance

Reply:
Nope, it's not stateful 🙂